### ISSUE TITLE
Proxy Setting in Arduino Pro IDE

### ISSUE THREAD

**Issue by TheSemiconductor** [arduino/arduino-pro-ide#410]
It would be nice to have an easy way to set a proxy in the ide. I know there's a way to do that in the CLI but i still haven't figured out how to do that.

**Comment by kittaakos** [comment]
Blocked by https://github.com/arduino/arduino-cli/issues/1161

**Comment by kittaakos** [comment]
@TheSemiconductor, could you please try the proxy settings with the [nightly](https://github.com/arduino/arduino-pro-ide#nightly-builds) build and let us know how it works? Thank you!

**Comment by kittaakos** [comment]
Implemented.
